crates/matrix-sdk-base/src/store/send_queue.rs

```diff
@@ -219,17 +225,23 @@ pub enum DependentQueuedRequestKind {
         key: String,
     },
 
-    /// Upload a file that had a thumbnail.
-    UploadFileWithThumbnail {
-        /// Content type for the file itself (not the thumbnail).
+    /// Upload a file or thumbnail depending on another file or thumbnail
+    /// upload.
+    #[serde(rename = "UploadFileWithThumbnail")]
```

Comment:
Thanks; i think we could use `serde(alias)` here, which is slightly better (will use the Rust name or the one specified in the alias, if the Rust name was missing).

```suggestion
    #[serde(alias = "UploadFileWithThumbnail")]
```